src/index.ts

```diff
@@ -6,6 +6,7 @@ import { container } from "tsyringe";
 import { CONFIG } from "./config";
 import "./injects";
 import { CommandService } from "./services";
+import { ActivityService } from "./services/activityService";
```

Comment:
It's just my preference, but you should export the activityService into the services/index.ts so it can be imported directly from "./services" like the command service.